### ISSUE TITLE
API no longer working

### ISSUE THREAD

**Issue by BeetleBonsai798** [snowby666/poe-api-wrapper#160]
Using the API gives the error:
ValueError: Bot chinchilla not found. Make sure the bot exists before creating new chat.

**Comment by mknight1300** [comment]
File "/usr/lib/python3.12/site-packages/poe_api_wrapper/api.py", line 690, in send_message
    raise e
  File "/usr/lib/python3.12/site-packages/poe_api_wrapper/api.py", line 660, in send_message
    raise ValueError(
ValueError: Bot gpt-4o not found. Make sure the bot exists before creating new chat.

**Comment by 7trail** [comment]
Can confirm that this is an issue for me as well. What's weird is that it still shows in poe.com that I've sent a request to make a message.

**Comment by phamxtien** [comment]
@snowby666
Today, it returns 
```
2024-06-13 06:26:45.351 | DEBUG    | poe_api_wrapper.api:connect_ws:164 - Failed to subscribe by sending SubscriptionsMutation. Raw response data: {'data': None, 'errors': [{'message': 'Server Error'}], 'extensions': {'is_final': True}}
2024-06-13 06:26:47.196 | DEBUG    | poe_api_wrapper.api:connect_ws:164 - Failed to subscribe by sending SubscriptionsMutation. Raw response data: {'data': None, 'errors': [{'message': 'Server Error'}], 'extensions': {'is_final': True}}
Rate limit exceeded for sending requests to poe.com. Please try again later.

```
It causes by in api.py
```
def subscribe(self):
        response_json = self.send_request('gql_POST', "SubscriptionsMutation", SubscriptionsMutation)
        if response_json['data'] == None and response_json["errors"]:
            raise RuntimeError(f'Failed to subscribe by sending SubscriptionsMutation. Raw response data: {response_json}')
```
and in utils.py
```
SubscriptionsMutation = {
    "subscriptions": [
                        {
                            "subscriptionName":"messageAdded",
                            "query":None,
                            "queryHash":"b739399cc69af7bb45a16c889a6ca6c24d3456337fde805ee7f480e6195a3eb7"
                        },
                        {
                            "subscriptionName":"messageCancelled",
                            "query":None,
                            "queryHash":"14647e90e5960ec81fa83ae53d270462c3743199fbb6c4f26f40f4c83116d2ff"
                        },
                        {
                            "subscriptionName":"messageDeleted",
                            "query":None,
                            "queryHash":"91f1ea046d2f3e21dabb3131898ec3c597cb879aa270ad780e8fdd687cde02a3"
                        },
                        {
                            "subscriptionName":"messageCreated",
                            "query":None,
                            "queryHash":"0445ef6a92aebf368dfb87659d2593b07a17d62ff48d0c0c8a0ae6ab2afd362c"
                        },
                        {
                            "subscriptionName":"viewerStateUpdated",
                            "query":None,
                            "queryHash":"cf9d367b40f59f4d087437912206dc0e3d344fb5085d0aba46cd88222a33edee"
                        },
                        {
                            "subscriptionName":"messageLimitUpdated",
                            "query":None,
                            "queryHash":"daec317b69fed95fd7bf1202c4eca0850e6a9740bc8412af636940227359a211"
                        },
                        {
                            "subscriptionName":"chatTitleUpdated",
                            "query":None,
                            "queryHash":"ee062b1f269ecd02ea4c2a3f1e4b2f222f7574c43634a2da4ebeb616d8647e06"
                        },
                        {
                            "subscriptionName":"knowledgeSourceUpdated",
                            "query":None,
                            "queryHash":"7de63f89277bcf54f2323008850573809595dcef687f26a78561910cfd4f6c37"
                        },
                        {
                            "subscriptionName":"messagePointLimitUpdated",
                            "query":None,
                            "queryHash":"94789388515b3c125c4a45d3c5112edffd102f8b72d4f152404f58e2aed9ec6d"
                        },
                        {
                            "subscriptionName":"chatMemberAdded",
                            "query":None,
                            "queryHash":"08b46c791cbb98b7e729c435d6b736ec60871dff58ec3a377ce818e30b50c1c8"
                        }
    ]
    
}
```
I don't know how to check if subscriptionName and queryHash change

**Comment by snowby666** [comment]
This is fixed in the v1.4.3. Please check documentation for new config.
